.babelrc

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
This file is being referenced by `awesome-typescript-loader` as well as `babel-loader` and should be kept